receiver/kubeletstatsreceiver/documentation.md

```diff
@@ -530,6 +530,14 @@ The time since the pod started
 | ---- | ----------- | ---------- | ----------------------- | --------- |
 | s | Sum | Int | Cumulative | true |
 
+### k8s.volume.used
```

Comment:
I know that currently the implementation follows the `k8s.volume.*` pattern, but since this metric is new I wonder if we should align it with the suggested SemConv [PR](https://github.com/open-telemetry/semantic-conventions/pull/2319) (if/when merged). It might feel weird to just have 1 metric on it's own pattern however it would be 1 less breaking change in the future.